models/reviews_model.js

```diff
@@ -1,9 +1,31 @@
 const db = require('../db/connection')
 
-exports.showReviews = () => {
-    return db.query(`SELECT * FROM reviews`)
+exports.showReviews = (category) => {
+    const validCategs = ['strategy', 'hidden-roles', 'dexterity', 'push-your-luck', 'roll-and-write', 'deck-building', 'engine-building']
```

Comment:
You need to dynamically check the database for what categories it currently has. 

At the moment, this will break on the front and you'll have to come back in two weeks to try and fix it then. 

Also, a further ticket is 'add category' which would also not work with this solution